.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
We can leave this to another PR, but the CI is not complete (we are missing the coverage and bench steps)